src/IDBStore.js

```diff
@@ -53,7 +53,7 @@
     IDBStore.getDB(dbName, function(error, db) {
       if (error) return callback(error);
       var transaction = db.transaction([IDBStore.DB_STORE_NAME], type);
-      transaction.onerror = function(e) {
+      transaction.onerror = (e) => {
         callback(this.error || 'unknown error');
```

Comment:
the use of `this` here seems dangerous, and could change with this PR I think, but I'm not sure if `onerror` gets bound anything for `this` or not.

Reply:
The use of `this` here looks a little suspicions .. should we calling the callback with `e` or `e.error` and not `this.error`? 

This change from back in 2015 looks a little suspect: a5279386047c43482c574a4a2a09508242ac0bb3

I will try and land this as a separate change.

Reply:
`event.target.error` seems to be way to go here.  Updated.

I can't seems to find any way to make these `onerror` callbacks happens since specifying bad database or a bad transaction type seems to throw rather than trigger `onerror`, so this is seems very hard to test.